dxr/vcs.py

```diff
@@ -45,11 +45,11 @@ def get_vcs_name(self):
         return type(self).__name__
 
     @classmethod
-    def invoke_vcs(cls, args, cwd):
+    def invoke_vcs(cls, args, **kwargs):
         """Return the result of invoking the VCS command on the repository,
-        with the current working directory set to the root directory.
+        with extra kwargs passed along to the Popen constructor.
         """
-        return subprocess.check_output([cls.command] + args, cwd=cwd)
+        return subprocess.check_output([cls.command] + args, **kwargs)
```

Comment:
Also, is there a reason to not make cwd required? It seems like asking for trouble to not specify it. We've already had that amply demonstrated, right?